webui/react/.prettierignore

```diff
@@ -0,0 +1,3 @@
+build
+.vscode
+./src/services/api-ts-sdk/*
```

Comment:
Ignore ProtoBuf generated files. What else do we need to ignore?

Reply:
maybe the same as eslintignore?